docs/node/task-queues.md

```diff
@@ -0,0 +1,58 @@
+---
+id: task-queues
+title: Task Queues in Node
+sidebar_label: Task Queues
+---
+
+## What is a Task Queue?
+
+import SharedTaskQueuesBasic from '../shared/task-queues-basic.md'
+
+<SharedTaskQueuesBasic
```

Comment:
Not strictly related to this PR but the first paragraph is inaccurate:
> From a high level, we can say that a Task Queue is exactly what the name suggests. It is a "first-in-first-out" queue for Tasks, where a Task is the context needed to execute a chunk of code that alters the "state" of a Workflow.

What about activity task queues? Those should be mentioned here as well IMHO.

Reply:
This one is exactly as written in the Go and Java docs, so I think we should discuss this issue separately. I agree that it's worth mentioning that Task Queues store both Tasks and Activities.